server/rest.py

```diff
@@ -116,196 +55,58 @@ def createTiles(self, item, params):
+                    '"%s" parameter is an incorrect type.' % paramName)
+        return tileSourceArgs
 
     @describeRoute(
-        Description('Remove a large image from this item.')
-        .param('itemId', 'The ID of the item.', paramType='path')
+        Description('Get large image metadata.')
+        .param('itemId', 'The ID of the item or "test".', paramType='path')
+        .errorResponse('ID was invalid.')
+        .errorResponse('Read access was denied for the item.', 403)
     )
-    @access.user
-    @loadmodel(model='item', map={'itemId': 'item'}, level=AccessType.WRITE)
-    def deleteTiles(self, item, params):
-        deleted = False
-        if 'largeImage' in item or 'largeImageJobId' in item:
-            job = None
-            if 'largeImageJobId' in item:
-                Job = self.model('job', 'jobs')
-                try:
-                    job = Job.load(item['largeImageJobId'], force=True,
-                                   exc=True)
-                except ValidationException:
-                    # The job has been deleted, but we still need to clean up
-                    # the rest of the tile information
-                    pass
-            if (item.get('expectedLargeImage') and job and
-                    job.get('status') in (
-                    JobStatus.QUEUED, JobStatus.RUNNING)):
-                # cannot cleanly remove the large image, since a conversion
-                # job is currently in progress
-                # TODO: cancel the job
-                # TODO: return a failure error code
-                return {
-                    'deleted': False
-                }
-
-            # If this file was created by the worker job, delete it
-            if 'largeImageJobId' in item:
-                if job:
-                    # TODO: does this eliminate all traces of the job?
-                    # TODO: do we want to remove the original job?
-                    Job.remove(job)
-                del item['largeImageJobId']
-
-            if 'largeImageOriginalId' in item:
-                # The large image file should not be the original file
-                assert item['largeImageOriginalId'] != item.get('largeImage')
-
-                if 'largeImage' in item:
-                    self.model('file').remove(self.model('file').load(
-                        id=item['largeImage'], force=True))
-                del item['largeImageOriginalId']
-
-            if item.get('largeImageSourceName'):
-                del item['largeImageSourceName']
-            if 'largeImage' in item:
-                del item['largeImage']
+    @access.public
+    def getTilesInfo(self, itemId, params):
+        if itemId == 'test':
+            item = 'test'
+            imageArgs = self._parseTestParams(params)
+        else:
+            item = self.model('item').load(
+                id=itemId, level=AccessType.READ,
+                user=self.getCurrentUser(), exc=True)
+            imageArgs = dict()
```

Comment:
This needs to be the params dict.  We support additional parameters for both getTilesInfo and getTile, such as asking for encoding='PNG' (the SVS tiles specifically support encoding, jpegSubsampling and jpegQuality as additional options).